lldb/unittests/Host/PipeTest.cpp

```diff
@@ -55,8 +55,6 @@ TEST_F(PipeTest, OpenAsReader) {
 }
 #endif
 
-// This test is flaky on Windows on Arm.
```

Comment:
Likely me that disabled it, but I doubt I had tested it on x86 Windows since we don't run a buildbot for it. Hopefully it'll be fine on WoA now.